travis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+dub test --compiler=${DC}
+
+if [ ! -z "$GH_TOKEN" ]; then
+    dub build -b docs --compiler=${DC}
+    cd docs
+    git init
+    git config user.name "Travis-CI"
+    git config user.email "travis@nodemeatspace.com"
+    git add .
+    git commit -m "Deployed to Github Pages"
+    #git push --force --quiet "https://${GH_TOKEN}@github.com/${TRAVIS_REPO_SLUG}" docs:gh-pages > /dev/null 2>&1
+    git push --force "https://${GH_TOKEN}@github.com/${TRAVIS_REPO_SLUG}" HEAD:gh-pages
```

Comment:
Make sure to suppress the output or your token might be shown in a bit error message.